packages/google_maps_flutter/google_maps_flutter_web/example/web/index.html

```diff
@@ -7,6 +7,7 @@
         <title>Browser Tests</title>
         <!-- This API key comes from: go/flutter-maps-web-tests-api-key (GCP project: flutter-infra) -->
         <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAa9cRBkhuxGq3Xw3HPz8SPwaVOhRmm7kk&libraries=geometry"></script>
+        <script src="https://unpkg.com/@googlemaps/markerclusterer@2.5.0/dist/index.min.js"></script>
```

Comment:
(If there's any change based on my comment on the app-facing package, we should make sure to make it here as well.)

Reply:
This is now updated to match the cdn used in web implementation as well.